src/AIModelCatalog.cpp

```
@@ -0,0 +1,648 @@
+}
+
+AIModelCatalog* AIModelCatalog::instance()
+{
+    if (!s_instance)
+        s_instance = new AIModelCatalog();
+    return s_instance;
+}
+
+AIModelCatalog* AIModelCatalog::qmlInstance(QQmlEngine*, QJSEngine*)
+{
+    return instance();
```

Cppcheck (2.21.0)

[error] There is an unknown macro here somewhere. Configuration is required. If slots is a macro then please configure it.

(unknownMacro)